packages/fxa-settings/src/components/images/index.stories.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import {
+  MailImage,
+  HeartsBrokenImage,
+  HeartsVerifiedImage,
+  RecoveryCodesImage,
+  TwoFactorAuthImage,
+  PreparedImage,
+} from './index';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'Components/Image',
+  component: PreparedImage,
+} as Meta;
+
+export const ImageWithCustomStyles = () => <MailImage className="w-1/4" />;
```

Comment:
Sweeet! Very helpful to have these in Storybook, seeing as vscode so helpfully shows the source code instead of the image.

Reply:
I'm going to try removing these -- I think they might be breaking the Storybook build 😬